config/default/manager_pod_disruption_budget.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
+    control-plane: controller-manager
+spec:
+  selector:
+    matchLabels:
+      app: azure-service-operator-v1
```

Comment:
Should use the same `matchLabels` as the Helm variant.

Reply:
Which should also probably be:
```
    matchLabels:
      control-plane: controller-manager
```

since that's what the deployment and various other matchers use.

We have an item to change this #4277, but for now I think best to keep them all in sync.

Reply:
Thanks, this was updated now!